_pages/en/capabilities.md

```diff
@@ -535,24 +537,87 @@ InfoSec supplies development teams and operational teams with an ample number of
 
 #### 3.4.2. Indicators  
 
-- What type of tools do you use to manage the work at various stages of a process?  
-  a. Card walls  
-  b. Storyboards  
-  c. Kanban boards  
-  d. We do not use any tools  
-  e. None of the above/not applicable  
+- My team uses visual displays and dashboards that are common in the context of software delivery to manage the work at various stages of a process  
+   a. Very Frequently (Ideal)  
+   b. Frequently  
+   c. Occasionally  
+   d. Rarely  
+   e. Very Rarely  
+   f. Never  
 
 - How long does it take to complete a project backlog?  
   a. More than I would expect  
   b. About right  
-  c. Less than I would expect  
+  c. Less than I would expect (Ideal)  
 
-- How do you gather and measure feedback from users?  
-  a. Manual testing, user acceptance testing or general acceptance criteria  
-  b. Direct email/phone/web contact form, web meetings, or chat room  
-  c. In-product qualitative feedback (e.g. Send-a-Smile) and survey such as Net Promoter Score (NPS)  
-  d. Usage telemetry is gathered with real user monitoring at 100% sampling  
-  e. None of the above/not applicable  
+- How often your team receives a feedback that created displays are complex, hard to understand, or do not provide actionable information?  
```

Comment:
- How often your team receives a feedback that created displays are complex, hard to understand, or do not provide actionable information?  
==>
- How often **does** your team receive feedback that displays are complex, hard to understand, or do not provide actionable information?